tutorials/haskell/golden-testing-in-haskell-with-hspec-gold/tutorial.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Golden testing in Haskell with Hspec-Golden
+published: 2016-08-10
+tags: haskell, golden testing, snapshot testing, expect testing
+libraries: hspec-golden-1.0.0
+language: haskell
+author: Adrian Vera
+author-name: Adrian Vera
+twitter-profile: adrian_med
+github-profile: medAdrian
+description: A golden test is an IO action that writes its result to a file. To pass the test, this output file should be identical to the corresponding golden file, which contains the correct result for the test.
```

Comment:
@MedAdrian This is the description of the library. However, I think we should focus here on the description of the tutorial. Probably:
```yaml
description:  In this tutorial we will write golden tests using hspec-golden. A golden test is an IO action that writes its result to a file.
```
I'm sure can come up with a better description than that :sweat_smile:

Reply:
I've updated the description to:

```
description: In this tutorial we will implement golden tests in Haskell using hspec-golden a super set of the known test framework hspec. A golden test is an Input/Output action that records the result of a test into a file known as the "Golden File".

Please let me know what do you think. 
```

Reply:
Looks good to me.